gradio/cli/commands/__init__.py

```diff
@@ -14,4 +15,5 @@
     "sketch",
     "upload_mcp",
     "hf_login",
+    "openaichat",
```

Comment:
I think `load-chat` would be a better name @jyotiradityadav ! That way we can add `load` command later as well.

Reply:
Let's go with `load chat`!